tee-worker/cli/src/base_cli/commands/litentry/id_graph_hash.rs

```diff
@@ -36,8 +36,14 @@ impl IDGraphHashCommand {
 		let mrenclave = direct_api.get_state_mrenclave().unwrap();
 		let shard = ShardIdentifier::decode(&mut &mrenclave[..]).unwrap();
 		let identity = Identity::from_did(self.did.as_str()).unwrap();
-		let id_graph_hash = direct_api.get_id_graph_hash(&shard, &identity).unwrap();
-		println!("{:?}", id_graph_hash);
+		match direct_api.get_id_graph_hash(&shard, &identity) {
```

Comment:
Why not `expect` ?
It can panic with more friendly message, additionaly `Ok` is not returned (there was a failure).

Reply:
What `Ok` is not returned?

Reply:
`unwrap` previously prevented from returning `Ok` from function.
After you fix, even if there is an error the function will return `Ok`